fastmri_recon/tests/training_scripts/unet_nc_train_test.py

```diff
@@ -7,7 +8,10 @@
 @pytest.mark.parametrize('kwargs',[
     {'dcomp': True},
 ])
-def test_train_unet_nc(create_full_fastmri_test_tmp_dataset, kwargs):
+@pytest.mark.parametrize('nufft_implementation',['tfkbnufft', 'tensorflow-nufft'])
+def test_train_unet_nc(create_full_fastmri_test_tmp_dataset, kwargs, nufft_implementation):
+    if nufft_implementation == 'tensorflow-nufft' and os.environ.get('CI', False) == True:
```

Comment:
```suggestion
    if nufft_implementation == 'tensorflow-nufft' and os.environ.get('CI', False):
```

Reply:
Ahhhh no, i got it wrong... It is:

`os.environ.get('CI', False) == "True"`

Reply:
and why do you do this ?

Reply:
Well environment variables are strings 
 So the Boolean value is stored as a string which is "True" or "False"

Reply:
can't you make it a `bool` when creating it? 

FYI:
```
bool('True')
```
returns 
```
True
```

Reply:
Also bool('False') returns True :P